spec/models/events/review_comment_spec.rb

```diff
@@ -26,6 +26,17 @@
 
 RSpec.describe Events::ReviewComment, type: :model do
   describe 'validations' do
+    subject { build :review_comment }
+
+    it 'is valid with valid attributes' do
+      expect(subject).to be_valid
+    end
+
+    it 'is not valid without github id' do
+      subject.github_id = nil
+      expect(subject).to_not be_valid
+    end
+
     it { is_expected.to belong_to(:pull_request) }
```

Comment:
This is a question.

This test skips its description. Is it a style convention or would

```
it 'belongs to a PullRequest' do
   is_expected.to belong_to(:pull_request)
end
```

be more declarative?